SLR.md

```diff
@@ -53,6 +53,7 @@ First page only
 ## Criteria
 ### Inclusion
 
+* all terms mentioned above must appear in the title
```

Comment:
~550 papers, maybe we should only select papers that contain one of our categories `["antipattern", "bad smell", "pattern"]`?